apps/api/src/app/controllers/command/__tests__/buildMockUuidGenerator.ts

```diff
@@ -0,0 +1,3 @@
+export default () => ({
+    generate: async () => Promise.resolve('9b1deb4d-3b7d-4bad-9bdd-2b0d7b3dcb6d'),
```

Comment:
This is just mock, but are we still thinking of using a coscrad prefix for our id's?